### ISSUE TITLE
Cannot access texture pack

### ISSUE THREAD

**Issue by RoboMWM** [sanjay900/PortalStickNew#4]
I'm not too interested in the texture pack command not working since I deliver my own resource pack to players, but I can't access the one designed for this plugin: http://tangentnetwork.tk/resourcepacks/Portal.zip

**Comment by sanjay900** [comment]
Yeah, that domain has changed. Your probably going to want to merge your
pack at some point later on because there's s bunch of additional textures
on the hoe, like 80 or something

**Comment by sanjay900** [comment]
https://tangentmc.net/resourcepacks/Portal.zip
